ivy_tests/test_ivy/helpers/testing_helpers.py

```diff
@@ -179,7 +179,13 @@ def _import_fn(fn_tree: str):
     fn_name = fn_tree[split_index + 1 :]
     module_to_import = fn_tree[:split_index]
     mod = importlib.import_module(module_to_import)
-    callable_fn = mod.__dict__[fn_name]
+    try:
+        callable_fn = mod.__dict__[fn_name]
+    except KeyError:
+        raise ImportError(
+            f"Unable to locate the function '{fn_name}' within '{module_to_import}'.\n"
+            "Ensure the function exists and is correctly spelled."
```

Comment:
one small request, can we also add a line, something around linking to the docs in case a function doesn't exist and they can contribute through a PR?
```
        + "Visit Ivy's open task page to learn more about contributing to the APIs! "
        + "https://lets-unify.ai/ivy/contributing/open_tasks.html\n"

```

Reply:
Sure! 

Can you suggest which one looks better to you?
```py
# suggestion 1
raise ImportError(
    f"Error: Function '{fn_name}' not found in module '{module_to_import}'.\n"
    "Ensure the correct function name and module path are specified.\n"
    "Interested in adding a new feature or function? Contribute to our project. For guidelines, visit:\n"
    "https://lets-unify.ai/ivy/contributing/open_tasks.html\n"
)

# suggestion 2
raise ImportError(
    f"Error: The function '{fn_name}' could not be found within the module '{module_to_import}'.\n"
    "Please double-check the function name and its associated path for accuracy.\n"
    "If this function is a new feature you'd like to see added, we'd love to hear from you! You can contribute to our project. For more details, please visit:\n"
    "https://lets-unify.ai/ivy/contributing/open_tasks.html\n"
)
```

Reply:
2 sounds good!
one small update: could we remove "for accuracy" and "added"? feel free to merge after that, thanks a lot!

Reply:
> 2 sounds good! one small update: could we remove "for accuracy" and "added"? feel free to merge after that, thanks a lot! @akshatvishu

Added the suggested changes. (I don't think I have the merge powers )